app/models/host.rb

```diff
@@ -12,7 +12,11 @@ def self.method_missing(method, *args, &block)
       end
     end
     if type.constantize.respond_to?(method, true)
-      type.constantize.send(method, *args, &block)
+      if method.to_s =~ /\Afind_in_(.*)\Z/ && args.size == 1 && args[0].is_a?(Hash)
```

Comment:
I see this is copied from elsewhere, but would this make sense?
```suggestion
      if method.to_s.start_with?('find_in_') && args.size == 1 && args[0].is_a?(Hash)
```